python/lsst/ci/prepare.py

```diff
@@ -28,6 +28,21 @@
 from future.utils import with_metaclass
 
 
+class RemoteError(Exception):
```

Comment:
please put a minimalist docstring here and described the parameters. git_errors is a list/tuple of exception objects?